BDArmory/UI/BDArmorySetup.cs

```diff
@@ -3238,11 +3242,22 @@ void WindowSettings(int windowID)
                                 SpawnUtils.HackActuators(vessel, BDArmorySettings.RUNWAY_PROJECT);
                             }
                         }
-                        if (HighLogic.LoadedSceneIsEditor && EditorLogic.fetch.ship is not null) GameEvents.onEditorShipModified.Fire(EditorLogic.fetch.ship);
+                        if (HighLogic.LoadedSceneIsEditor)
+                        {
+                            if (EditorLogic.fetch.ship is not null) GameEvents.onEditorShipModified.Fire(EditorLogic.fetch.ship);
+                            if (BDArmorySettings.RUNWAY_PROJECT && BDAEditorArmorWindow.Instance) BDAEditorArmorWindow.Instance.SetupLegalityValues();
+                        }
                     }
                     if (CompSettings.CompOverridesEnabled)
-                        BDArmorySettings.COMP_CONVENIENCE_CHECKS = GUI.Toggle(SRightRect(line), BDArmorySettings.COMP_CONVENIENCE_CHECKS, StringUtils.Localize("Use AI/WM Overrides"));
-
+                    {
+                        if (BDArmorySettings.COMP_CONVENIENCE_CHECKS != (BDArmorySettings.COMP_CONVENIENCE_CHECKS = GUI.Toggle(SLeftRect(++line), BDArmorySettings.COMP_CONVENIENCE_CHECKS, StringUtils.Localize("#LOC_BDArmory_Settings_CompChecks"))))//Runway Project
```

Comment:
Is there a reason this got moved to the left on the next line? It seems a bit out-of-place there:
![Screenshot_20250301_211220](https://github.com/user-attachments/assets/6a365cf5-ef3c-4d92-a231-bb35c9da4264)

Reply:
Oops. That's a mistake.